app/forms/candidate_interface/pick_course_form.rb

```diff
@@ -64,7 +68,7 @@ def unique_name_for(course)
     end
 
     def available_courses
-      @available_courses ||= courses_for_current_cycle.exposed_in_find.includes(:accredited_provider, :course_options).order(:name)
+      @available_courses ||= GetAvailableCoursesForProvider.new(provider).call
     end
 
     def courses_with_names
```

Comment:
I decided to use this pick course form from the old implementation only to get the data for the course field. It is kind a tech debt but I would like to hear the opinions of the team on this @DFE-Digital/find-and-apply-devs

Reply:
I'm not sure what you mean, it does the same thing as far as I can tell and it's being used [here](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/forms/candidate_interface/continuous_applications/which_course_are_you_applying_to_step.rb#L15) as well. Makes sense to sue the same method to get available courses.